config/deployments/drainer.toml

```diff
@@ -0,0 +1,39 @@
+[drainer]
+loop_interval = 500
+max_read_count = 100
+num_partitions = 64
+shutdown_interval = 1000
+stream_name = "drainer_stream"
+
+[kms]
+key_id = "kms_key_id"
+region = "kms_region"
+
+[log.console]
+enabled = "true"
+level = "DEBUG"
+log_format = "json"
+
+[log.telemetry]
+metrics_enabled = true
+otel_exporter_otlp_endpoint = "http://localhost:4317"
+
+[master_database]
+dbname = "master_database_name"
+host = "http://localhost:4000"
```

Comment:
The host should contain only `localhost`, no scheme (`http://`) or port.
When connecting to the database, the database connection string will be constructed, which uses the `postgres://` or `postgresql://` scheme.